### ISSUE TITLE
Multiselect selected array argument improperly handles numeric strings ["1"]

### ISSUE THREAD

**Issue by scottgrayson** [LaravelCollective/html#500]
```blade
<h1>alpha string values</h1>
{!! Form::select("alpha[]", ["a" => "selected", "b" => "not"], ["a"], ["multiple" => true]) !!}

<h1>numeric string values</h1>
{!! Form::select("numeric[]", ["1" => "selected", "2" => "not"], ["1"], ["multiple" => true]) !!}
```
### produces
![image](https://user-images.githubusercontent.com/7796074/37183377-bc3f7e2a-2303-11e8-922d-a6d017f22936.png)

### expected
["1" => "selected"] option to be selected in second multiselect

### other notes
When values are loaded from url
ex: `?numeric[]=1`
`Request::input("numeric")` result is an array of string 
`[0 => "1"]`
so there is no option to use integers instead of strings

**Comment by scottgrayson** [comment]
I am using version 5.3.2

**Comment by robovarga** [comment]
I mean it was fixed there -> #501
update package :) and should be fine!
